components/tabs/src/schema.rs

```diff
@@ -34,10 +31,15 @@ const CREATE_META_TABLE_SQL: &str = "
 const CREATE_PENDING_REMOTE_DELETE_TABLE_SQL: &str = "
     CREATE TABLE IF NOT EXISTS pending_remote_tab_closures (
```

Comment:
Shall we make this name more generic, since we're doing a migration—maybe `remote_tabs_commands`?